src/utils/duration-to-human.ts

```diff
@@ -13,29 +13,34 @@ import { pluralize } from './pluralize';
  * Examples:
  *   durationToHuman(Duration.fromObject({days:5, hours:2,minutes:1, seconds:30})) -> "5 days 2 hours"
  *   durationToHuman(Duration.fromObject({hours:2,minutes:1, seconds:30})) -> "2 hours 1 minute"
- *   durationToHuman(Duration.fromObject({minutes:10, seconds:30})) -> "10 minutes"
+ *   durationToHuman(Duration.fromObject({minutes:10, seconds:30})) -> "10 minutes, 30 seconds"
  *   durationToHuman(Duration.fromObject({seconds:30})) -> "30 seconds"
  */
-export default function durationToHuman(duration: Duration): string {
-  let out = '';
-
+export function durationToHuman(duration: Duration): string {
   duration = duration.shiftTo('days', 'hours', 'minutes', 'seconds');
   let days = duration.get('days');
   let minutes = duration.get('minutes');
   let hours = duration.get('hours');
   let seconds = duration.get('seconds');
 
+  let parts = [];
```

Comment:
This will be inferred as an `any[]`. Might make sense to explicitly mark it as `string[]` for nicer DX purposes.

Reply:
👍 done!